components/Buttons/src/MDCButton.m

```diff
@@ -215,6 +215,12 @@ - (void)encodeWithCoder:(NSCoder *)aCoder {
   [aCoder encodeObject:_accessibilityLabelForState forKey:MDCButtonAccessibilityLabelsKey];
 }
 
++ (void)initialize {
+  // Default background colors.
+  [[MDCButton appearance] setBackgroundColor:MDCColorFromRGB(MDCButtonDefaultBackgroundColor)
```

Comment:
It is because MDCButton does not call commonMDCButtonInit for vanilla init. (we dont override it)

Even we use the vanilla init in our [navbar buttons](https://github.com/material-components/material-components-ios/blob/develop/components/ButtonBar/src/private/MDCAppBarButtonBarBuilder.m#L100)